### PR TITLE
BAU Update images used by testscontainers for M1 compatibility

### PR DESCRIPTION
**Pull request by nlsteers**
Updated the following images used by `testcontainers`:
-   `testcontainersofficial/ryuk` -> `testcontainers/ryuk:0.3.3`
-  `oribio16/alpine-sqs` -> `mvisonneau/alpine-sqs:1.2.0`

### CODE REVIEW

**Comment by katstevens** [comment]
Am I right that this image is only used in the pact tests?

**Comment by nlsteers** [comment]
These images are used by any tests that require `testcontainers`, e.g. anything that needs to test Postgres / SQS interactions.

File: src/test/resources/testcontainers.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# Digest of image testcontainersofficial/ryuk:latest for linux/amd64 from tags list on https://hub.docker.com/r/testcontainersofficial/ryuk
-ryuk.container.image=testcontainersofficial/ryuk@sha256:8fdd715bf34975d088273bcea497fe048f9b768ba4c7fc690ab6e4ddc995e317
+# arm64/amd64 friendly ryuk image
+ryuk.container.image=testcontainers/ryuk:0.3.3
```
Comment (katstevens) [inline src/test/resources/testcontainers.properties:2]: Is there any difference between `testcontainers` and `testcontainersofficial`? They both seem up to date 🤔

Comment (nlsteers) [reply src/test/resources/testcontainers.properties:2]: afaict, the difference is that the former vends multi platform images whereas the latter does not...

**Review by katstevens — APPROVED** [review]
OK, let's try this out 👍

**Pull request merged**